### PR TITLE
Feature/build website

### PR DESCRIPTION
**Pull request by martin-cervantes**
:heavy_check_mark: Add asynchronous communication with promises or async/await
:heavy_check_mark: Using weather API from openweathermap.org
:heavy_check_mark: Change units form metric to imperial

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Weather_App",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (Torres-ssf) [inline package.json]: - [x] Great job adding Webpack to your project. However, it's best practice to create at least 2 scripts for your application, one for development and another one for production. We need to also to inform in which [mode](https://webpack.js.org/configuration/mode/) Webpack will run in these scripts. We will have something like this:

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
  },
```

- [x] We also need to inform users about these script in the README file. This information can be placed in the Usage section. This way users will know how they can use your application.

**OPTIONAL**

- [x]  You can create an even better script for development stages. You will need to add the `
webpack-dev-server` and create a script like this:
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
Running the `start` script will automatically open the index file in the browser, without the need to open it manually

**Review by Torres-ssf — APPROVED** [review]
### Changes required :muscle:
Hi there @mcervantes71 :hand: 
Nice job on this project. But we need to make some changes before moving on.

- [x] As it said in the instructions, your app should allow users to search for a specific location.
Screenshot from 2020-08-18 10-44-49

- [x] We also need to change the app appearance based on the receive forecast.
Screenshot from 2020-08-18 11-17-13

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Comment by martin-cervantes1** [comment]
:heavy_check_mark: The app allows users to search for a specific location.
:heavy_check_mark: Change the app appearance based on the receive data.
:heavy_check_mark: Add information about script in the README file

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED

Hi @mcervantes71 this looks good :+1: kudos on making changes, everything works fine now, please merge and move on **This is approved** 

**RAPHAEL NORIODE** :mask: STAY_SAFE

**Pull request merged**